Sources/SpeziOnboarding/OnboardingDataSource.swift

```diff
@@ -49,12 +49,12 @@ public class OnboardingDataSource: Module, EnvironmentAccessible {
     /// Adds a new exported consent form represented as `PDFDocument` to the ``OnboardingDataSource``.
     ///
     /// - Parameter consent: The exported consent form represented as `ConsentDocumentExport` that should be added.
-    public func store(_ consent: PDFDocument, identifier: String = ConsentDocumentExport.Defaults.documentIdentifier) async throws {
+    public func store(_ consent: ConsentDocumentExport) async throws {
```

Comment:
Sounds good! I added it accordingly. Since the ConsentDocumentExport requires to provide the markdown data and the ExportConfiguration, we would need to provide some placeholders when creating it inside the "old" store function. But I think this should be uncritical since we have a cached PDF file anyways, and the export function will never be called.